rules/anti-behavior/random_behavior.yara

```diff
@@ -43,6 +45,7 @@ rule java_random: low {
 rule rand_call: medium {
   meta:
     description = "exhibits random behavior"
+    filetypes   = "text/x-c,text/x-php,text/x-perl"
```

Comment:
With verbose filetypes like these, it's less likely that folks will willingly update this list for rules. Can we use the file extension associated to the MIME type instead? Such as: `filetypes = "c,php,p"` for this example?

Most folks don't have MIME types memorized, whereas they are likely to have file extensions memorized. For example, `.exe` vs [application/vnd.microsoft.portable-executable](https://www.iana.org/assignments/media-types/application/vnd.microsoft.portable-executable)

Even if someone has a MIME type memorized, there is often unclear standards on which to use. For example, for `.zip` there are 3 MIME types in use :(

Reply:
Good point. This was the easiest since the map was set up that way, but perhaps we can flip that around and make this cleaner.

Reply:
Swapped over to referencing extensions in [`24b9bf2` (#898)](https://github.com/chainguard-dev/malcontent/pull/898/commits/24b9bf2d1a69e487c5a5df8fcb33fcbf022be7eb).